tests/tests_geomstats/test_visualization_spd.py

```diff
@@ -0,0 +1,66 @@
+"""Unit tests for visualization."""
```

Comment:
Rename file to test_visualization_spd_matrices.py
Change top docsstring to ... for visualization of SPD matrices